lib/brainstem/concerns/controller_dsl.rb

```diff
@@ -564,6 +566,16 @@ def valid_params_tree(requested_context = action_name.to_sym)
           .to_h
           .inject(ActiveSupport::HashWithIndifferentAccess.new) do |hsh, (field_name_proc, field_config)|
 
+          if field_config.has_key?(:if)
```

Comment:
Could use the [Safe Assignment in Condition idiom](https://github.com/rubocop-hq/ruby-style-guide#safe-assignment-in-condition) to reduce a line:

```
if (condition = field_config[:if])
  case condition
  ...
end
```

Reply:
Nice call. I was just thinking the same thing.